src/main/java/com/statful/client/VertxMetricsImpl.java

```diff
@@ -95,7 +90,7 @@ public boolean isEnabled() {
 
     @Override
     public void eventBusInitialized(final EventBus bus) {
-        this.customMetricsConsumer = new CustomMetricsConsumer(bus, this.getOrCreateSender(), statfulMetricsOptions);
+        new CustomMetricsConsumer(bus, this.getOrCreateSender(), statfulMetricsOptions);
```

Comment:
We could continue to keep this reference mostly for a reminder that we should implement more method from the parent class. e.g.: close which should then stop that collector, along with the other collectors.

Reply:
👍